### PR TITLE
Initial project

### PR DESCRIPTION
**Pull request by corpglory-dev**
It requires Grafana 6.3.3+

You'll get `Object prototype may only be an Object or null: undefined` error in Grafana versions  < 6.3.3

## Screenshots
Config editor:
image

Generated timeseries:
image

### CODE REVIEW

File: build/webpack.base.conf.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+const path = require('path');
+const webpack = require('webpack');
+const moment = require('moment');
+
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const ReplaceInFileWebpackPlugin = require('replace-in-file-webpack-plugin');
+
+function resolve(dir) {
```
Comment (jonyrock) [inline build/webpack.base.conf.js]: hey, checkout another way to "resolve": https://github.com/CorpGlory/express-webpack-example/blob/master/build/webpack.base.conf.js#L13

File: package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "name": "grafana-datasource-react-ts-webpack-template",
+  "version": "1.0.0",
+  "description": "Some description",
+  "main": "dist/module.js",
+  "scripts": {
+    "build": "webpack --config build/webpack.prod.conf.js",
+    "dev": "webpack --config build/webpack.dev.conf.js"
+  },
+  "keywords": [],
+  "author": "CorpGlory",
```
Comment (jonyrock) [inline package.json]:
```suggestion
  "author": "CorpGlory Inc.",
```

File: src/datasource.ts [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+export class TemplateDatasource extends DataSourceApi<TemplateQuery> {
+
+  type: string;
+  url: string;
+  name: string;
+  headers: any;
+  withCredentials: boolean;
+
+  /** @ngInject */
```
Comment (jonyrock) [inline src/datasource.ts:20]: oh, it's still angular inject , right?

Comment (rozetko) [reply src/datasource.ts:20]: Yes, React is not supported for `DatasourceApi` yet

**Pull request merged**